testsuite/forge-cli/src/main.rs

```diff
@@ -330,6 +330,7 @@ fn get_changelog(prev_commit: Option<&String>, upstream_commit: &str) -> String
 fn get_test_suite(suite_name: &str) -> ForgeConfig<'static> {
     match suite_name {
         "land_blocking_compat" => land_blocking_test_compat_suite(),
+        "land_blocking_real" => land_blocking_real_suite(),
```

Comment:
nit: `land_blocking_real_suite` -> `land_blocking_release_suite` may be?

Reply:
Good point -- I just got rid of some old tests that we haven't verified with the new setup yet, and made this the `pre_release` suite.